src/ProducerEventProcessor/UploaderClientCaptureEventCollectorTest.cpp

```diff
@@ -82,14 +82,13 @@ class UploaderClientCaptureEventCollectorTest : public testing::Test {
     produce_finished_ = true;
   }
 
+  static constexpr size_t kUploadBufferSize = 100;
```

Comment:
Nit: `constexpr` implies `const` and `const` implies `static`. So technically the `static` is not needed.

Reply:
Thanks for the suggestion! 

But due to "non-static data member cannot be constexpr", I might keep `static` here.

Reply:
I think you're confusing the different usages of `static`. `static` means something different depending on where it appears: in a class declaration, in a function body, on the global level. What you're citing only refers to "non-static data members", meaning when `static` is used in a class declaration.

Reply:
I just realized that this is in an anonymoys class. Never mind. Sorry. :see_no_evil:

Reply:
Update: Moved `kUploadBufferSize` outside of the class and removed `static` for this and other constants.